tachyon/math/elliptic_curves/msm/algorithms/icicle/icicle_msm_bls12_381_g2.cc

```diff
@@ -55,8 +65,7 @@ bool IcicleMSM<bls12_381::G2AffinePoint>::Run(
     ::bls12_381::g2_projective_t ret;
     gpuError_t error = tachyon_bls12_381_g2_msm_cuda(
         reinterpret_cast<const ::bls12_381::scalar_t*>(&cpu_scalars[start_idx]),
-        reinterpret_cast<const ::bls12_381::g2_affine_t*>(
-            &bases[start_idx]),
+        reinterpret_cast<const ::bls12_381::g2_affine_t*>(&bases[start_idx]),
```

Comment:
It looks like there is a diff even though there are no changes. I need to check the linter.